angular-drag-and-drop-lists.js

```diff
@@ -409,6 +417,10 @@
           if (!data) return stopDragover();
         }
 
+        if (typeof data === "object") {
```

Comment:
This change is new. Whats is this for?

Reply:
Safety check to ensure an empty/null data doesn't cause an error. I observed it happening.